SampleTests: commuteSearchSampleTest failed

This test failed!

To configure my behavior, see [the Build Cop Bot documentation](https://github.com/googleapis/repo-automation-bots/tree/master/packages/buildcop).

If I'm commenting on this issue too often, add the `buildcop: quiet` label and
I will stop commenting.

---

commit: 7b872c3b28ec3505c4a4efe9d9af41baea4b97cc
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/b2e1535e-cbe9-4f20-ad83-b638b2b95655), [Sponge](http://sponge2/b2e1535e-cbe9-4f20-ad83-b638b2b95655)
status: failed

```
------------------------------------------------------------
- testing jobs/v3
------------------------------------------------------------
[ERROR] SampleTests.commuteSearchSampleTest  Time elapsed: 11.758 s  <<< FAILURE!
expected to contain a match for:
    .*matchingJobs.*commuteInfo.*
but was:
    Company generated: GenericData{classInfo=[careerSiteUri, derivedInfo, displayName, eeoText, externalId, headquartersAddress, hiringAgency, imageUri, keywordSearchableJobCustomAttributes, name, size, suspended, websiteUri], {displayName=Google, externalId=company:5933863803961527561, headquartersAddress=1600 Amphitheatre Parkway Mountain View, CA 94043}}
    Company created: {"derivedInfo":{"headquartersLocation":{"latLng":{"latitude":37.42231,"longitude":-122.0846241},"locationType":"STREET_ADDRESS","postalAddress":{"addressLines":["1600 Amphitheatre Pkwy, Mountain View, CA 94043, USA"],"administrativeArea":"CA","locality":"Mountain View","postalCode":"94043","regionCode":"US"},"radiusInMiles":7.391152814933662E-5}},"displayName":"Google","externalId":"company:5933863803961527561","headquartersAddress":"1600 Amphitheatre Parkway Mountain View, CA 94043","name":"projects/java-docs-samples-testing/companies/5b128583-d70e-4f4e-9f0a-7683951ee8cd"}
    Job generated: GenericData{classInfo=[addresses, applicationInfo, companyDisplayName, companyName, compensationInfo, customAttributes, degreeTypes, department, derivedInfo, description, employmentTypes, incentives, jobBenefits, jobEndTime, jobLevel, jobStartTime, languageCode, name, postingCreateTime, postingExpireTime, postingPublishTime, postingRegion, postingUpdateTime, processingOptions, promotionValue, qualifications, requisitionId, responsibilities, title, visibility], {applicationInfo=GenericData{classInfo=[emails, instruction, uris], {uris=[http://careers.google.com]}}, companyName=projects/java-docs-samples-testing/companies/5b128583-d70e-4f4e-9f0a-7683951ee8cd, description=Design, develop, test, deploy, maintain and improve software., requisitionId=jobWithRequiredFields:-7495153024510409260, title=Software Engineer}}
    Job created: {"addresses":["1600 Amphitheatre Parkway, Mountain View, CA 94043"],"applicationInfo":{"uris":["http://careers.google.com"]},"companyDisplayName":"Google","companyName":"projects/java-docs-samples-testing/companies/5b128583-d70e-4f4e-9f0a-7683951ee8cd","derivedInfo":{"locations":[{"latLng":{"latitude":37.42231,"longitude":-122.0846241},"locationType":"STREET_ADDRESS","postalAddress":{"addressLines":["1600 Amphitheatre Pkwy, Mountain View, CA 94043, USA"],"administrativeArea":"CA","locality":"Mountain View","postalCode":"94043","regionCode":"US"},"radiusInMiles":7.391152814933662E-5}]},"description":"Design, develop, test, deploy, maintain and improve software.","languageCode":"en","name":"projects/java-docs-samples-testing/jobs/126472165399634630","postingCreateTime":"2020-04-12T20:41:25.366Z","postingExpireTime":"2020-05-12T20:41:25.358Z","postingPublishTime":"2020-04-12T20:41:25.416Z","postingUpdateTime":"2020-04-12T20:41:25.366Z","requisitionId":"jobWithRequiredFields:-7495153024510409260","title":"Software Engineer","visibility":"ACCOUNT_ONLY"}
    {"metadata":{"requestId":"0de2e24c-4fda-468c-9b74-d6f8b6154ee3:APAb7ITWqAJaqFYracYw8YxpivJj+d5Xjg=="}}
    Job deleted
    Company deleted
    
	at SampleTests.commuteSearchSampleTest(SampleTests.java:107)

[ERROR] Failures: 
[ERROR]   SampleTests.commuteSearchSampleTest:107 expected to contain a match for:
    .*matchingJobs.*commuteInfo.*
but was:
    Company generated: GenericData{classInfo=[careerSiteUri, derivedInfo, displayName, eeoText, externalId, headquartersAddress, hiringAgency, imageUri, keywordSearchableJobCustomAttributes, name, size, suspended, websiteUri], {displayName=Google, externalId=company:5933863803961527561, headquartersAddress=1600 Amphitheatre Parkway Mountain View, CA 94043}}
    Company created: {"derivedInfo":{"headquartersLocation":{"latLng":{"latitude":37.42231,"longitude":-122.0846241},"locationType":"STREET_ADDRESS","postalAddress":{"addressLines":["1600 Amphitheatre Pkwy, Mountain View, CA 94043, USA"],"administrativeArea":"CA","locality":"Mountain View","postalCode":"94043","regionCode":"US"},"radiusInMiles":7.391152814933662E-5}},"displayName":"Google","externalId":"company:5933863803961527561","headquartersAddress":"1600 Amphitheatre Parkway Mountain View, CA 94043","name":"projects/java-docs-samples-testing/companies/5b128583-d70e-4f4e-9f0a-7683951ee8cd"}
    Job generated: GenericData{classInfo=[addresses, applicationInfo, companyDisplayName, companyName, compensationInfo, customAttributes, degreeTypes, department, derivedInfo, description, employmentTypes, incentives, jobBenefits, jobEndTime, jobLevel, jobStartTime, languageCode, name, postingCreateTime, postingExpireTime, postingPublishTime, postingRegion, postingUpdateTime, processingOptions, promotionValue, qualifications, requisitionId, responsibilities, title, visibility], {applicationInfo=GenericData{classInfo=[emails, instruction, uris], {uris=[http://careers.google.com]}}, companyName=projects/java-docs-samples-testing/companies/5b128583-d70e-4f4e-9f0a-7683951ee8cd, description=Design, develop, test, deploy, maintain and improve software., requisitionId=jobWithRequiredFields:-7495153024510409260, title=Software Engineer}}
    Job created: {"addresses":["1600 Amphitheatre Parkway, Mountain View, CA 94043"],"applicationInfo":{"uris":["http://careers.google.com"]},"companyDisplayName":"Google","companyName":"projects/java-docs-samples-testing/companies/5b128583-d70e-4f4e-9f0a-7683951ee8cd","derivedInfo":{"locations":[{"latLng":{"latitude":37.42231,"longitude":-122.0846241},"locationType":"STREET_ADDRESS","postalAddress":{"addressLines":["1600 Amphitheatre Pkwy, Mountain View, CA 94043, USA"],"administrativeArea":"CA","locality":"Mountain View","postalCode":"94043","regionCode":"US"},"radiusInMiles":7.391152814933662E-5}]},"description":"Design, develop, test, deploy, maintain and improve software.","languageCode":"en","name":"projects/java-docs-samples-testing/jobs/126472165399634630","postingCreateTime":"2020-04-12T20:41:25.366Z","postingExpireTime":"2020-05-12T20:41:25.358Z","postingPublishTime":"2020-04-12T20:41:25.416Z","postingUpdateTime":"2020-04-12T20:41:25.366Z","requisitionId":"jobWithRequiredFields:-7495153024510409260","title":"Software Engineer","visibility":"ACCOUNT_ONLY"}
    {"metadata":{"requestId":"0de2e24c-4fda-468c-9b74-d6f8b6154ee3:APAb7ITWqAJaqFYracYw8YxpivJj+d5Xjg=="}}
    Job deleted
    Company deleted
    
[ERROR] Tests run: 12, Failures: 1, Errors: 0, Skipped: 0
```

--- a/jobs/v3/src/main/java/com/google/samples/CommuteSearchSample.java
+++ b/jobs/v3/src/main/java/com/google/samples/CommuteSearchSample.java
@@ -44,7 +44,7 @@ public final class CommuteSearchSample {
 
   // [START commute_search]
 
-  public static void commuteSearch(String companyName) throws IOException {
+  public static void commuteSearch(String companyName) throws IOException, InterruptedException {
     // Make sure to set the requestMetadata the same as the associated search request
     RequestMetadata requestMetadata =
         new RequestMetadata()
@@ -77,6 +77,7 @@ public final class CommuteSearchSample {
     SearchJobsResponse response =
         talentSolutionClient.projects().jobs().search(DEFAULT_PROJECT_ID, searchJobsRequest)
             .execute();
+    Thread.sleep(5000);
     System.out.println(response);
   }
   // [END commute_search]

--- a/jobs/v3/src/main/java/com/google/samples/CustomAttributeSample.java
+++ b/jobs/v3/src/main/java/com/google/samples/CustomAttributeSample.java
@@ -90,7 +90,8 @@ public final class CustomAttributeSample {
   /**
    * CustomAttributeFilter on String value CustomAttribute
    */
-  public static void filtersOnStringValueCustomAttribute() throws IOException {
+  public static void filtersOnStringValueCustomAttribute()
+      throws IOException, InterruptedException {
     // Make sure to set the requestMetadata the same as the associated search request
     RequestMetadata requestMetadata =
         new RequestMetadata()
@@ -117,6 +118,7 @@ public final class CustomAttributeSample {
             .jobs()
             .search(DEFAULT_PROJECT_ID, searchJobsRequest)
             .execute();
+    Thread.sleep(1000);
     System.out.println(response);
   }
   // [END custom_attribute_filter_string_value]
@@ -126,7 +128,7 @@ public final class CustomAttributeSample {
   /**
    * CustomAttributeFilter on Long value CustomAttribute
    */
-  public static void filtersOnLongValueCustomAttribute() throws IOException {
+  public static void filtersOnLongValueCustomAttribute() throws IOException, InterruptedException {
     // Make sure to set the requestMetadata the same as the associated search request
     RequestMetadata requestMetadata =
         new RequestMetadata()
@@ -153,6 +155,7 @@ public final class CustomAttributeSample {
             .jobs()
             .search(DEFAULT_PROJECT_ID, searchJobsRequest)
             .execute();
+    Thread.sleep(1000);
     System.out.println(response);
   }
   // [END custom_attribute_filter_long_value]
@@ -162,7 +165,7 @@ public final class CustomAttributeSample {
   /**
    * CustomAttributeFilter on multiple CustomAttributes
    */
-  public static void filtersOnMultiCustomAttributes() throws IOException {
+  public static void filtersOnMultiCustomAttributes() throws IOException, InterruptedException {
     // Make sure to set the requestMetadata the same as the associated search request
     RequestMetadata requestMetadata =
         new RequestMetadata()
@@ -190,6 +193,7 @@ public final class CustomAttributeSample {
             .jobs()
             .search(DEFAULT_PROJECT_ID, searchJobsRequest)
             .execute();
+    Thread.sleep(1000);
     System.out.println(response);
   }
   // [END custom_attribute_filter_multi_attributes]

--- a/jobs/v3/src/main/java/com/google/samples/EmailAlertSearchSample.java
+++ b/jobs/v3/src/main/java/com/google/samples/EmailAlertSearchSample.java
@@ -41,7 +41,7 @@ public final class EmailAlertSearchSample {
   /**
    * Search jobs for alert.
    */
-  public static void searchForAlerts(String companyName) throws IOException {
+  public static void searchForAlerts(String companyName) throws IOException, InterruptedException {
     // Make sure to set the requestMetadata the same as the associated search request
     RequestMetadata requestMetadata =
         new RequestMetadata()
@@ -66,6 +66,7 @@ public final class EmailAlertSearchSample {
             .jobs()
             .searchForAlert(DEFAULT_PROJECT_ID, request)
             .execute();
+    Thread.sleep(1000);
     System.out.println(response);
   }
   // [END search_for_alerts]

--- a/jobs/v3/src/main/java/com/google/samples/FeaturedJobsSearchSample.java
+++ b/jobs/v3/src/main/java/com/google/samples/FeaturedJobsSearchSample.java
@@ -69,7 +69,8 @@ public final class FeaturedJobsSearchSample {
   // [START search_featured_job]
 
   /** Searches featured jobs. */
-  public static void searchFeaturedJobs(String companyName) throws IOException {
+  public static void searchFeaturedJobs(String companyName)
+      throws IOException, InterruptedException {
     // Make sure to set the requestMetadata the same as the associated search request
     RequestMetadata requestMetadata =
         new RequestMetadata()
@@ -98,6 +99,7 @@ public final class FeaturedJobsSearchSample {
             .jobs()
             .search(DEFAULT_PROJECT_ID, searchJobsRequest)
             .execute();
+    Thread.sleep(1000);
     System.out.println(response);
   }
   // [END search_featured_job]

--- a/jobs/v3/src/main/java/com/google/samples/GeneralSearchSample.java
+++ b/jobs/v3/src/main/java/com/google/samples/GeneralSearchSample.java
@@ -63,7 +63,8 @@ public final class GeneralSearchSample {
   /**
    * Simple search jobs with keyword.
    */
-  public static void basicSearcJobs(String companyName, String query) throws IOException {
+  public static void basicSearcJobs(String companyName, String query)
+      throws IOException, InterruptedException {
     // Make sure to set the requestMetadata the same as the associated search request
     RequestMetadata requestMetadata =
         new RequestMetadata()
@@ -92,6 +93,7 @@ public final class GeneralSearchSample {
             .jobs()
             .search(DEFAULT_PROJECT_ID, searchJobsRequest)
             .execute();
+    Thread.sleep(1000);
 
     System.out.println(searchJobsResponse);
   }
@@ -103,7 +105,7 @@ public final class GeneralSearchSample {
    * Search on category filter.
    */
   public static void categoryFilterSearch(String companyName, List<String> categories)
-      throws IOException {
+      throws IOException, InterruptedException {
     // Make sure to set the requestMetadata the same as the associated search request
     RequestMetadata requestMetadata =
         new RequestMetadata()
@@ -131,6 +133,7 @@ public final class GeneralSearchSample {
             .jobs()
             .search(DEFAULT_PROJECT_ID, searchJobsRequest)
             .execute();
+    Thread.sleep(1000);
 
     System.out.println(searchJobsResponse);
   }
@@ -142,7 +145,7 @@ public final class GeneralSearchSample {
    * Search on employment types.
    */
   public static void employmentTypesSearch(String companyName, List<String> employmentTypes)
-      throws IOException {
+      throws IOException, InterruptedException {
     // Make sure to set the requestMetadata the same as the associated search request
     RequestMetadata requestMetadata =
         new RequestMetadata()
@@ -170,6 +173,7 @@ public final class GeneralSearchSample {
             .jobs()
             .search(DEFAULT_PROJECT_ID, searchJobsRequest)
             .execute();
+    Thread.sleep(1000);
 
     System.out.println(searchJobsResponse);
   }
@@ -183,7 +187,7 @@ public final class GeneralSearchSample {
    * "{year}-{month}-{day}T{hour}:{min}:{sec}[.{frac_sec}]Z" e.g. "2017-01-15T01:30:15.01Z"
    */
   public static void dateRangeSearch(String companyName, String startTime, String endTime)
-      throws IOException {
+      throws IOException, InterruptedException {
     // Make sure to set the requestMetadata the same as the associated search request
     RequestMetadata requestMetadata =
         new RequestMetadata()
@@ -216,6 +220,7 @@ public final class GeneralSearchSample {
             .jobs()
             .search(DEFAULT_PROJECT_ID, searchJobsRequest)
             .execute();
+    Thread.sleep(1000);
 
     System.out.println(searchJobsResponse);
   }
@@ -227,7 +232,7 @@ public final class GeneralSearchSample {
    * Search on language codes.
    */
   public static void languageCodeSearch(String companyName, List<String> languageCodes)
-      throws IOException {
+      throws IOException, InterruptedException {
     // Make sure to set the requestMetadata the same as the associated search request
     RequestMetadata requestMetadata =
         new RequestMetadata()
@@ -255,6 +260,7 @@ public final class GeneralSearchSample {
             .jobs()
             .search(DEFAULT_PROJECT_ID, searchJobsRequest)
             .execute();
+    Thread.sleep(1000);
 
     System.out.println(searchJobsResponse);
   }
@@ -266,7 +272,7 @@ public final class GeneralSearchSample {
    * Search on company display name.
    */
   public static void companyDisplayNameSearch(String companyName, List<String> companyDisplayNames)
-      throws IOException {
+      throws IOException, InterruptedException {
     // Make sure to set the requestMetadata the same as the associated search request
     RequestMetadata requestMetadata =
         new RequestMetadata()
@@ -294,6 +300,7 @@ public final class GeneralSearchSample {
             .jobs()
             .search(DEFAULT_PROJECT_ID, searchJobsRequest)
             .execute();
+    Thread.sleep(1000);
 
     System.out.println(searchJobsResponse);
   }
@@ -304,7 +311,8 @@ public final class GeneralSearchSample {
   /**
    * Search on compensation.
    */
-  public static void compensationSearch(String companyName) throws IOException {
+  public static void compensationSearch(String companyName)
+      throws IOException, InterruptedException {
     // Make sure to set the requestMetadata the same as the associated search request
     RequestMetadata requestMetadata =
         new RequestMetadata()
@@ -347,6 +355,7 @@ public final class GeneralSearchSample {
             .jobs()
             .search(DEFAULT_PROJECT_ID, searchJobsRequest)
             .execute();
+    Thread.sleep(1000);
 
     System.out.println(searchJobsResponse);
   }

--- a/jobs/v3/src/main/java/com/google/samples/HistogramSample.java
+++ b/jobs/v3/src/main/java/com/google/samples/HistogramSample.java
@@ -44,7 +44,7 @@ public final class HistogramSample {
   /**
    * Histogram search
    */
-  public static void histogramSearch(String companyName) throws IOException {
+  public static void histogramSearch(String companyName) throws IOException, InterruptedException {
     // Make sure to set the requestMetadata the same as the associated search request
     RequestMetadata requestMetadata =
         new RequestMetadata()
@@ -81,6 +81,7 @@ public final class HistogramSample {
             .jobs()
             .search(DEFAULT_PROJECT_ID, searchJobsRequest)
             .execute();
+    Thread.sleep(1000);
 
     System.out.println(searchJobsResponse);
   }

--- a/jobs/v3/src/main/java/com/google/samples/LocationSearchSample.java
+++ b/jobs/v3/src/main/java/com/google/samples/LocationSearchSample.java
@@ -54,7 +54,7 @@ public final class LocationSearchSample {
    * Basic location Search
    */
   public static void basicLocationSearch(String companyName, String location, double distance)
-      throws IOException {
+      throws IOException, InterruptedException {
     // Make sure to set the requestMetadata the same as the associated search request
     RequestMetadata requestMetadata =
         new RequestMetadata()
@@ -79,6 +79,7 @@ public final class LocationSearchSample {
             .setSearchMode("JOB_SEARCH");
     SearchJobsResponse response =
         talentSolutionClient.projects().jobs().search(DEFAULT_PROJECT_ID, request).execute();
+    Thread.sleep(1000);
     System.out.println(response);
   }
   // [END basic_location_search]
@@ -90,7 +91,7 @@ public final class LocationSearchSample {
    */
   public static void keywordLocationSearch(String companyName, String location, double distance,
       String keyword)
-      throws IOException {
+      throws IOException, InterruptedException {
     // Make sure to set the requestMetadata the same as the associated search request
     RequestMetadata requestMetadata =
         new RequestMetadata()
@@ -117,6 +118,7 @@ public final class LocationSearchSample {
             .setSearchMode("JOB_SEARCH");
     SearchJobsResponse response =
         talentSolutionClient.projects().jobs().search(DEFAULT_PROJECT_ID, request).execute();
+    Thread.sleep(1000);
     System.out.println(response);
   }
   // [END keyword_location_search]
@@ -126,7 +128,8 @@ public final class LocationSearchSample {
   /**
    * City location Search
    */
-  public static void cityLocationSearch(String companyName, String location) throws IOException {
+  public static void cityLocationSearch(String companyName, String location)
+      throws IOException, InterruptedException {
     // Make sure to set the requestMetadata the same as the associated search request
     RequestMetadata requestMetadata =
         new RequestMetadata()
@@ -150,6 +153,7 @@ public final class LocationSearchSample {
             .setSearchMode("JOB_SEARCH");
     SearchJobsResponse response =
         talentSolutionClient.projects().jobs().search(DEFAULT_PROJECT_ID, request).execute();
+    Thread.sleep(1000);
     System.out.println(response);
   }
   // [END city_location_search]
@@ -161,7 +165,7 @@ public final class LocationSearchSample {
    */
   public static void multiLocationsSearch(String companyName, String location1, double distance1,
       String location2)
-      throws IOException {
+      throws IOException, InterruptedException {
     // Make sure to set the requestMetadata the same as the associated search request
     RequestMetadata requestMetadata =
         new RequestMetadata()
@@ -187,6 +191,7 @@ public final class LocationSearchSample {
             .setSearchMode("JOB_SEARCH");
     SearchJobsResponse response =
         talentSolutionClient.projects().jobs().search(DEFAULT_PROJECT_ID, request).execute();
+    Thread.sleep(1000);
     System.out.println(response);
   }
   // [END multi_locations_search]
@@ -197,7 +202,7 @@ public final class LocationSearchSample {
    * Broadening location Search
    */
   public static void broadeningLocationsSearch(String companyName, String location)
-      throws IOException {
+      throws IOException, InterruptedException {
     // Make sure to set the requestMetadata the same as the associated search request
     RequestMetadata requestMetadata =
         new RequestMetadata()
@@ -222,6 +227,7 @@ public final class LocationSearchSample {
             .setSearchMode("JOB_SEARCH");
     SearchJobsResponse response =
         talentSolutionClient.projects().jobs().search(DEFAULT_PROJECT_ID, request).execute();
+    Thread.sleep(1000);
     System.out.println(response);
   }
   // [END broadening_location_search]

